src/resolvers/user.js

```diff
@@ -68,27 +68,23 @@ export default {
     updateUser: combineResolvers(
       isAuthenticated,
       async (parent, { username }, { models, me }) => {
-        const user = await models.User.findById(me.id);
-        return await user.update({ username });
+        console.log({models: models, me})
+        return  await models.User.findByIdAndUpdate(me.id, { username: username}, {new: true});
       },
     ),
 
     deleteUser: combineResolvers(
       isAdmin,
       async (parent, { id }, { models }) => {
-        return await models.User.destroy({
-          where: { id },
-        });
+        return await models.User.findByIdAndRemove(id).then(() => true)
```

Comment:
What does `.then(() => true)` do?

Reply:
Maybe as returning value for the consuming function, which then can return a proper message to client ?
If that's actually the case i admit i would use a more explicit variable than a simple bool XD

Reply:
yeah true. I use an explicit value because in the graphql schema output value of the delete mutation is Bool while the resolved promise is a query object.